lib/charms/opensearch/v0/opensearch_base_charm.py

```diff
@@ -300,24 +305,35 @@ def _on_peer_relation_departed(self, event: RelationDepartedEvent):
         else:
             event.defer()
 
-    def _on_opensearch_data_storage_detaching(self, _: StorageDetachingEvent):
+    def _on_opensearch_data_storage_detaching(self, _: StorageDetachingEvent):  # noqa
```

Comment:
please be more specific, like
```suggestion
    def _on_opensearch_data_storage_detaching(self, _: StorageDetachingEvent):  # noqa: E501
```